src/StructureFactors.jl

```diff
@@ -931,3 +931,75 @@ function sf_slice(sf::StructureFactor, points::Vector;
     return slice
 end
 
+
+
+#========== Powder Averaging WIP ==========#
+""" gaussian_smoothing(k, q, S, L)
+    Returns 'Powder-Averaged' contribution to different bins according to 
+the exact formula for Gaussian Smoothing Kernel for each S(q). 
+
+Args 
+    k :: Range of 'bins' for Powder Averaging. 
+    q :: Magnitude of 'q' vector; Gaussian smoothing only a function of magnitude. 
+    S :: Intensity at 'q' vector. 
+    L :: Length-scale for smoothing kernel; suggested maximum(Lx, Ly, Lz). 
+        User can control this but issues may arise if σ for Gaussian Smoothing kernel is too big or small w.r.t spacing between q-vectors. 
+"""
+function gaussian_smoothing(k, q, S, L)
+    sig2 = (2π/L)^2.0 
```

Comment:
Probably better to do `x^2` because then Julia has a better chance of recognizing it's a "simple" exponential.